src/rules/at-import-no-partial-extension/index.js

```diff
@@ -23,6 +23,13 @@ export default function (on, options) {
     })
     if (!validOptions) { return }
 
+    result.warn((
```

Comment:
We'd normally just commented out all the tests and then we'd [add a test or two for the deprecation warning](https://github.com/stylelint/stylelint/blob/master/src/rules/font-family-name-quotes/__tests__/index.js#L269). That felt like the best balance between moving the project forward and using our time wisely.